src/components/CreateNewProjectWizard/Gatsby/SelectStarterDialog.js

```diff
@@ -0,0 +1,302 @@
+A starter object from starter.yml contains the following data:
+[
+   {
+      "url": "https://wonism.github.io/",
+      "repo": "https://github.com/wonism/gatsby-advanced-blog",
+      "description": "n/a",
+      "tags": [
+        "Portfolio",
+        "Redux"
+      ],
+      "features": [
+        "feature items"
+      ]
+  }, ... 
+]
+*/
+class SelectStarterDialog extends PureComponent<Props, State> {
+  state = {
+    loading: true,
+    starters: [],
+    selectedStarterInModal: '',
+    paginationIndex: 10,
+    filterString: '',
+  };
+
+  PAGINATION_STEP = 10;
+
+  static getDerivedStateFromProps(nextProps: Props, prevState: State) {
+    // Clear search string on modal open display
+    return {
+      ...prevState,
+      filterString: '',
+      selectedStarterInModal: nextProps.selectedStarter,
+    };
+  }
+
+  componentDidMount() {
+    fetch(
+      'https://raw.githubusercontent.com/gatsbyjs/gatsby/master/docs/starters.yml'
+    )
+      .then(response => response.text())
+      .then(yamlText => {
+        const starters = yaml.safeLoad(yamlText);
+
+        this.setState({
+          loading: false,
+          starters,
+        });
+      });
+  }
+
+  prepareUrlForCodesandbox(repoUrl: string) {
+    // Remove http protocol
+    const sandboxUrl = `https://codesandbox.io/s/${repoUrl.replace(
+      /(^\w+:|^)\/\//,
+      ''
+    )}`;
+    // Remove .com from github.com --> to have /s/github/repo
+    return sandboxUrl.replace(/\.com/, '');
+  }
+
+  handleDialogOK = () => {
+    this.props.updateFieldValue(
+      'projectStarter',
+      this.state.selectedStarterInModal
+    );
+
+    this.setState({
+      selectedStarterInModal: '',
+    });
+    this.props.hideModal();
+  };
+
+  setStarter = starter => {
+    this.setState({
+      selectedStarterInModal: starter,
+    });
+  };
+
+  handleShowMore = () => {
+    let newIndex = this.state.paginationIndex + this.PAGINATION_STEP;
+    newIndex = Math.min(this.state.starters.length, newIndex); // limit
+    this.setState({
+      paginationIndex: newIndex,
+    });
+  };
+
+  updateSearchString = evt => {
+    this.setState({
+      filterString: evt.target.value,
+    });
+  };
+
+  render() {
+    const { isVisible, hideModal } = this.props;
+    const {
+      loading,
+      starters,
+      selectedStarterInModal,
+      paginationIndex,
+      filterString,
+    } = this.state;
+    const filteredStarters = starters.filter(
+      starter => filterString === '' || starter.repo.includes(filterString)
+    );
+    const disabledUseSelect = selectedStarterInModal === '';
+
+    return (
+      <Modal isVisible={isVisible} onDismiss={hideModal}>
+        <ModalHeader title="Select starter" />
+
+        <MainContent>
+          <Paragraph>
+            For a better overview you can also have a look at the Gatsby
+            starters library{' '}
+            <ExternalLink href="https://www.gatsbyjs.org/starters/">
+              here.
+            </ExternalLink>

```

Comment:
I think it's slightly more conventional to make the link inline:

```
For a better overview you can also have a look at the{' '}
<ExternalLink href="https://www.gatsbyjs.org/starters/">
  Gatsby starters library
</ExternalLink>.

```

Reply:
OK, modified. 
What I've noticed with the external links in the summary pane is that the link is not fully clickable. I think that's also happening on master so this is not introduced here. 
Also the link visiblity is pretty bad as it's not visible that it's clickable. What do you think would it be OK to add a thin light rounded box around these links work?
Or maybe underlining would also work similar to the `import project` button.

For the hover I need to check what's causing it - looks like the width of the container is not wide enough. I haven't found the cause yet.

Here is how this looks:
![screenrecording_externallink_hover](https://user-images.githubusercontent.com/3046542/48999812-077f4f00-f158-11e8-85d8-06b30b0d9073.gif)

Reply:
Hover issue fixed with PR #337. It was an issue with the backface pointer events. 
We could improve styling later - thin outline not added.